routing/routing_benchmarks/bicycle_routing_tests.cpp

```diff
@@ -45,7 +45,7 @@ UNIT_CLASS_TEST(BicycleTest, Smoke)
   TestRouters(start, final);
 }
 
-UNIT_CLASS_TEST(BicycleTest, MailRuOfficeToUlSimanova5K2)
```

Comment:
Our search engine is able to search by lat-lons, routers expect start and final points as a couple of doubles, not text descriptions.